Makefile

```diff
@@ -178,10 +179,7 @@ unit-test-coverage:
 
 .PHONY: vendor
 vendor:
-	@hash dep > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		$(GO) get -u github.com/golang/dep/cmd/dep; \
-	fi
-	dep ensure -vendor-only
+	GO111MODULE=on go mod vendor
```

Comment:
We could maybe do a `go mod tidy` to clear removed deps.

Reply:
Done.